### ISSUE TITLE
CAT-Seg's training setting

### ISSUE THREAD

**Issue by BuRr-Lee** [wusize/CLIPSelf#4]
i want to know how do you train the CAT-Seg model after replacing its clip with eva (trained using clipself method).
especially, do you freeze the swin transformer backbone? do you finetune only attention layers in clip?

**Comment by wusize** [comment]
Hi, we use the model from OpenAI for the results on Cat-Seg. The training of Cat-Seg remain unchanged, where attention layers of CLIP are finetuned.

**Comment by BuRr-Lee** [comment]
you mean you first train the **original OpenAI-pretrained CLIP (instead of EVA-CLIP)** model with CLIPSelf method, then apply it to CAT-Seg?

**Comment by wusize** [comment]
> you mean you first train the **original OpenAI-pretrained CLIP (instead of EVA-CLIP)** model with CLIPSelf method, then apply it to CAT-Seg?

Yeah. We need to make sure the comparison with Cat-Seg is under the fair setting.
